app/services/induction/amend_participant_cohort.rb

```diff
@@ -123,12 +123,23 @@ def current_induction_record_changed?
       end
     end
 
+    def fip_historical_induction_programme_for(school_cohort, lead_provider_id, delivery_partner_id)
+      partnership = Partnership.create_with(relationship: true)
```

Comment:
Could we add two tests to ensure we check the following:
- defaults to an existing partnership which **isn't a relationship** if it exists for a "transferred" participants between two providers in that cohort
- otherwise creates a new relationship partnership for that provider in that cohort
just to make sure we're explicit in testing this?

Reply:
Done